ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py

```diff
@@ -978,3 +978,49 @@ def test_tensorflow_gather_nd(
         indices=indices,
         batch_dims=batch_dims,
     )
+
+
+# transpose
+@st.composite
+def _get_perm_helper(draw):
+    shape = draw(st.shared(helpers.get_shape(min_num_dims=1), key="shape"))
+    dimensions = [x for x in range(len(shape))]
+    perm = draw(st.permutations(dimensions))
+    return perm
+
+
+@handle_frontend_test(
+    fn_tree="tensorflow.transpose",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shape=st.shared(helpers.get_shape(min_num_dims=1), key="shape"),
+    ),
+    perm=_get_perm_helper(),
+    conjugate=st.booleans(),
+)
+def test_tensorflow_general_functions_transpose(
```

Comment:
While not incorrect, this can just be written as as test_tensorflow_transpose instead of test_tensorflow_general_functions_transpose to stay in line with the rest of the tests